.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @aws/sagemakerexperimentsadmin
```

Comment:
can we add some comments similar to this one: https://github.com/aws/aws-cdk/blob/master/.github/CODEOWNERS

Reply:
adde comment.

We can add users if necessary later, but for now everyone under weixun should be a member of
sagemakerexperimentsadmin. Ask one of the maintainers to be added https://github.com/orgs/aws/teams/sagemakerexperimentsadmin/members